tests/test_buffer_dynamic.py

```diff
@@ -208,8 +293,59 @@ def test_headroomOverride(self, dvs, testlog):
         # remove lossless PG 3-4 on interface
         dvs.runcmd("config interface buffer priority-group lossless remove Ethernet0 3-4")
 
-    def test_bufferModel(self, dvs, testlog):
+    def test_mtuUpdate(self, dvs, testlog):
         self.setup_db(dvs)
 
-        metadata = self.config_db.get_entry("DEVICE_METADATA", "localhost")
-        assert metadata["buffer_model"] == "traditional"
+        test_mtu = '1500'
+        default_mtu = '9100'
+        expectedProfileMtu = self.make_lossless_profile_name(self.originalSpeed, self.originalCableLen, mtu = test_mtu)
+        expectedProfileNormal = self.make_lossless_profile_name(self.originalSpeed, self.originalCableLen)
+
+        # update the mtu on the interface
+        dvs.runcmd("config interface mtu Ethernet0 {}".format(test_mtu))
+
+        # configure lossless PG 3-4 on interface
+        dvs.runcmd("config interface buffer priority-group lossless add Ethernet0 3-4")
+
+        self.app_db.wait_for_entry("BUFFER_PG_TABLE", "Ethernet0:3-4")
+        self.app_db.wait_for_entry("BUFFER_PROFILE_TABLE", expectedProfileMtu)
+        self.check_new_profile_in_asic_db(dvs, expectedProfileMtu)
+        self.app_db.wait_for_field_match("BUFFER_PG_TABLE", "Ethernet0:3-4", {"profile": "[BUFFER_PROFILE_TABLE:{}]".format(expectedProfileMtu)})
+
+        dvs.runcmd("config interface mtu Ethernet0 {}".format(default_mtu))
+
+        self.app_db.wait_for_deleted_entry("BUFFER_PROFILE_TABLE", expectedProfileMtu)
```

Comment:
check in asic db for profile deleted?

Reply:
No, we can’t. Similar as the previous one.